lib/runtime-core/src/parse.rs

```diff
@@ -82,7 +82,7 @@ pub fn read_module<
 
         func_assoc: Map::new(),
         signatures: Map::new(),
-        backend: MCG::backend_id(),
+        backend: MCG::backend_id().to_string(),
```

Comment:
If we know that it has to be `&'static str`, then we can change it to be a `&'static str` in the structs too and remove all uses of `to_string()`!